internal/bminventory/inventory.go

```diff
@@ -1153,7 +1153,7 @@ func (b *bareMetalInventory) updateExternalImageInfo(infraEnv *common.InfraEnv,
 	}
 
 	var version string
-	if osImage.Version != nil {
+	if osImage.OpenshiftVersion != nil {
```

Comment:
it probably mean that it wasn't covered in tests

Reply:
Yeah, I saw this and couldn't resist fixing it. Want me to add tests for it in this PR or I could move this fix and the tests to a followup?

Reply:
followup would be fine.